zingo-testutils/src/lib.rs

```diff
@@ -653,7 +625,121 @@ pub mod scenarios {
         )
     }
 
-    pub async fn two_wallet_one_miner_fund_transparent(
+    pub async fn faucet_recipient_default() -> (
```

Comment:
And then change this to just be `faucet_recipient`..  that way when I am writing my test I am (by default) just writing `faucet_recipient`..  and only typing extra characters in special cases where I need to specify the network upgrades...

Reply:
see my other comment